include/arch/arm/arch/object/vcpu.h

```diff
@@ -52,6 +52,9 @@ struct gicVCpuIface {
     uint32_t hcr;
     uint32_t vmcr;
     uint32_t apr;
+    /* virq_t[] requires word-size alignment; add extra padding for
+     * 64-bit platforms to make this struct packed. */
+    uint32_t gicVCpuIface_padding;
```

Comment:
Let's keep the field's name simple:
```suggestion
    uint32_t _padding;
```

Reply:
`The underscore prefix is reserved for functions and types used by the compiler and standard library.`
I'd rather use some macro that generates a unique and irrelevant name if I were forced to.
For a long while the C parser did not support underscore identifiers at all for this reason. There is no need to go doing this unnecessarily, and I object to "let's keep the field's name simple". No. Let's go shopping instead.

Reply:
Yes, definitely not with underscore. We could call it just `padding` without underscore, though, at least there is precedent for that in device code: https://github.com/seL4/seL4/blob/d06d281ff245a8d4bd3a322926e4186cf2d342a5/include/drivers/irq/omap3.h#L28-L33

and in the `reply` struct: https://github.com/seL4/seL4/blob/d06d281ff245a8d4bd3a322926e4186cf2d342a5/include/object/structures.h#L396-L397

I'm not 100% sure about how the disambiguation between field names in the C parser will come out, but we won't be referring to the field explicitly, so it should not make anything more ugly.

Reply:
Yeah, the seL4 coding conventions say "Prefix struct fields with the name of the struct to avoid namespace conflicts in the proof. (Much existing kernel code does not follow this directive; new code should adopt the recommended practice to avoid namespace collisions in proofs.)" and presumably there was a reason for this. Not sure if it matters for padding fields that don't show up in the state relation, but with small PRs like this I'm not keen to add new flavours to the soup.

Reply:
Ok, given that bit in the style guide, let's stick with your original name. We should investigate if that would make a difference, but that can be a separate thing. The padding name we remove in the other struct was also not just `padding`, so we're not doing anything new.